signalfx-java/src/main/java/com/signalfx/signalflow/StreamRequestException.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2019 SignalFx, Inc. All rights reserved.
+ */
+package com.signalfx.signalflow;
+
+import java.util.List;
+
+/**
+ * Exception thrown when the computation failed after being started.
```

Comment:
That seems like copy-pasta; this exception is for errors requesting the computation.

Reply:
More that I am not sure what to call it. 😄